src/Docs/CLI/Evaluate.hs

```diff
@@ -215,7 +215,7 @@ runSearch term = do
       , ("start", Just "1")
       , ("hoogle", Just $ Text.encodeUtf8 $ Text.pack term)
       ]
-  res <- fetch req
+  res <- fetchHttp req
```

Comment:
These requests need to be cached. Let's use `fetch` here.

Reply:
Oh yes, of course we need to cache that. But I have instead changed fetchHttp to do the caching, as suggested below :)